cli/run.go

```diff
@@ -115,16 +116,35 @@ func runAndExit(bitriseConfig models.BitriseDataModel, inventoryEnvironments []e
 
 	// Run selected configuration
 	if buildRunResults, err := runWorkflowWithConfiguration(startTime, workflowToRunID, bitriseConfig, inventoryEnvironments); err != nil {
```

Comment:
Do not we want to move the `Submitting anonymized usage information...` logs to the beginning of the build? So that the last lines of the log are only about the build status

Reply:
I'm fine with it being at the end. At least in this scope.